### DISCUSSION TITLE
run script in windows (solved)

### DISCUSSION THREAD

**Discussion by jpsala** [johnlindquist/kit#1216] [General]
Hi, is there a way to run a script from another windows app? in this case, it would be autohotkey, scriptkit plus Autohotkey = extasis :)

**Comment by johnlindquist** [comment]
@jpsala As long as the other app can write a file:

Write the full path to your script to the file: `~/.kit/run.txt`
![CleanShot 2023-04-27 at 17 57 04](https://user-images.githubusercontent.com/36073/235014597-b28efa30-2eda-4f3b-bf54-2d2c478b4be6.png)

For example, you can see how it works from the Script Kit VS code extension:

https://github.com/johnlindquist/kit-extension/blob/main/src/extension.ts#L81

The app watches the `run.txt` file for changes. Once it detects a change, it attempts to read the file for the script and will run it.

Make sense?

**Reply by jpsala** [reply]
It's working perfect @johnlindquist!!!
Thank you man!
Your software is great
